source/_integrations/zwave_js.markdown

```diff
@@ -826,9 +826,9 @@ After ensuring you are using an extension cable, perform a network heal.
 
 The combination of these two steps corrects a large number of reported difficulties.
 
-#### I have an Aeotec Gen5 controller, and it isn't detected on my Raspberry Pi 4?
+#### I have an Aeotec Gen5 controller, and it isn't detected on my Raspberry Pi&nbsp;4?
 
-The first-generation Gen5 controller has a known bug when plugged into a Pi 4 and possibly other systems. Aeotec released the Gen5+ stick to correct this bug. Gen5 users can plug their sticks into a USB 2.0 hub in order to overcome the issue.
+The first-generation Gen5 controller has a known bug when plugged into a Pi&nbsp;4 and possibly other systems. Aeotec released the Gen5+ stick to correct this bug. Gen5 users can plug their sticks into a USB&nbsp;2.0 hub in order to overcome the issue.
```

Comment:
Why is the html needed?

Reply:
a non-breaking space to avoid a line break before the number. It happened on my system.